### PR TITLE
Documentation: Fixes #8812 - All news are published 8 may 23

### LINKED ISSUE #8812: All news are published 8 may 23
All news items have this date - instead it should take the timestamp from the Markdown filename.

### PR DESCRIPTION
**Pull request by hubertfilho**
Dates for v2.10 and v2.11 from Twitter and v2.12 from Linkedin.

### CODE REVIEW

File: readme/news/20230508-release-2-10.md [hunk 1]
```diff
@@ -67,4 +68,4 @@ Our mobile app now has an improved search function that performs text searches i
 
 ## Improved filesystem sync performance
 
-Thanks to the hard work of jd1378, the sync no longer freezes during filesystem synchronisation. We know how frustrating that can be, and we're thrilled to have solved this issue. Getting filesystem sync to work on Android is never easy due to the restrictions put in place by Google, especially since they frequently change, but we're committed to delivering the best possible experience for our Android users.
\ No newline at end of file
+Thanks to the hard work of jd1378, the sync no longer freezes during filesystem synchronisation. We know how frustrating that can be, and we're thrilled to have solved this issue. Getting filesystem sync to work on Android is never easy due to the restrictions put in place by Google, especially since they frequently change, but we're committed to delivering the best possible experience for our Android users.
```
Comment (hubertfilho) [inline readme/news/20230508-release-2-10.md:71]: I don't know why there's this change. I didn't touch this line.

Comment (hubertfilho) [reply readme/news/20230508-release-2-10.md:71]: And it doesn't make sense to remove it. Because, it already was there.

Comment (laurent22) [reply readme/news/20230508-release-2-10.md:71]: Can you please remove it though?

**Comment by laurent22** [comment]
Fixed in 0eeec7f300ab0f52bb56b668fe103fb72cfc1b32

**Pull request closed without merging**